adam/experiment/__init__.py

```diff
@@ -197,7 +223,20 @@ def execute_experiment(
             except OSError:
                 logging.warning("Cannot log learner state to %s", str(learner_path))
                 log_learner_state = False
-                logging.warning("Proceeding without logging learner state")
+                logging.warning(
+                    "Proceeding without logging learner state. The experiment results from the observers won't be valid if this is not logged."
```

Comment:
I find this slightly confusing as "this" would seem to refer to the learner state and it's not clear why that would make the observer results invalid except for when the experiment is resumed. Maybe something like this?
```suggestion
                    "Proceeding without logging learner state. The experiment results from the observers won't be valid if the experiment is resumed without a log of the learner state."
```